### PR TITLE
Register tmasikt.is-a.dev

### PR DESCRIPTION
**Pull request by tmasikt**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

A-Record to ip `194.190.169.59`

### CODE REVIEW

**Review by Stef-00012 — APPROVED** [review]
seems good to me

File: domains/mail.tmasikt.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "tmasikt's mail record"
```
Comment (Stef-00012) [inline domains/mail.tmasikt.json:2]: nevermind, missing a comma `,` after the description

File: domains/tmasikt.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "tmasikt's server record"
```
Comment (Stef-00012) [inline domains/tmasikt.json:2]: missing a comma `,` after the description

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

File: domains/tmasikt.json (hunk 2 above)
Comment (tmasikt) [reply domains/tmasikt.json]: fixed

**Review by Stef-00012 — APPROVED** [review]
seems good now

**Pull request merged**